### PR TITLE
Add splitpart filters

### PR DESCRIPTION
**Pull request by nixikanius**
Provided methods needed to get part of splitted string in one step. In course of http://stackoverflow.com/questions/30515456/split-string-into-list-in-jinja

### CODE REVIEW

File: split.py [hunk 1]
```diff
@@ -7,16 +7,24 @@ def split_string(string, seperator=' '):
     except Exception, e:
         raise errors.AnsibleFilterError('split plugin error: %s, string=%s' % str(e),str(string) )
 
-def split_regex(string, seperator_pattern):
+def split_regex(string, seperator_pattern='\s+'):
```
Comment (timraasveld) [inline split.py]: Good idea, makes sense to have a default parameter like in `split`

File: split.py [hunk 2]
```diff
@@ -7,16 +7,24 @@ def split_string(string, seperator=' '):
...
     try:
         return re.split(seperator_pattern, string)
     except Exception, e:
-        raise errors.AnsibleFilterError('split plugin error: %s' % str(e))
+        raise errors.AnsibleFilterError('split plugin error: %s, string=%s' % str(e),str(string) )
```
Comment (timraasveld) [inline split.py]: `'split plugin error: %s, string: %s'`

**Comment by timraasveld** [comment]
I feel that most programmers will instantly know what `string | split[0]` does, but not `string | splitpart`, and therefore close this PR.

Your changes to the default parameter and error string are very welcome though, if you could change your PR or open a new one that'd be great!

File: split.py (hunk 2 above)
Comment (nixikanius) [inline split.py]: Just copied from your code above. :)

**Comment by nixikanius** [comment]
If `string | split[0]` worked I would not be created this PR. Unfortunately your clause does not work, we get:

> fatal: [deb7bare]: FAILED! => {"failed": true, "msg": "template error while templating string: expected token 'end of print statement', got '['. String: {{item|split[0]}}"}

So to get one part of split we need to do 1 excess task (as in [StackOverflow solution](http://stackoverflow.com/a/30517735/2941371)). To prevent this I introduce this PR because have not found a way to solve the issue in one task.

**Comment by nixikanius** [comment]
Oh, I'm sorry - I forgot about magic parentheses! So worked sample will be `(string | split)[0]`. This PR could be closed.

**Pull request closed without merging**